package.json

```diff
@@ -131,7 +132,8 @@
     "bootstrap-sass": "3.3.7",
     "bowser": "^1.7.1",
     "bundle-loader": "^0.5.4",
-    "cbioportal-frontend-commons": "0.2.6",
+    "cbioportal-frontend-commons": "^0.2.7",
+    "cbioportal-ts-api-client": "^1.0.0-beta.0",
```

Comment:
Current API version in [swagger](https://www.cbioportal.org/api/swagger-ui.html) doc is 1.0 (beta)